db/migrate/20191219153625_change_planning_raise_salary_to_string_in_employees.rb

```diff
@@ -0,0 +1,8 @@
+class ChangePlanningRaiseSalaryToStringInEmployees < ActiveRecord::Migration[6.0]
+  def up
```

Comment:
No problem with defining `up` and `down`, this is explicit and I know what this migration will do in both directions.  One other option would be to define `change` and do a single `change_column :employees, :planning_raise_salary, :string`, and I believe `ActiveRecord` will automatically figure out the reverse migration.

Reply:
The information I found said that just defining change would make the migration irreversible, though it could have been out of date (or just wrong).

Reply:
Ah, you're right.  I thought this was one of the ones that did auto-reverse, but I'm mistaken.  👍